aip/general/0162.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: 162
-state: draft
+state: approved
```

Comment:
I'd prefer to separate "we're still making changes" from "it's ready from approval".

Reply:
Perhaps set to `reviewing`?